localized/games/eggquiz/eggquizliteracy_levels_hi.tsv

```diff
@@ -938,70 +938,70 @@ SpecialTest_21	1	210001, 210002, 210003, 210004, 210005
 			210001	soundonly_sentence					She is reading under the table.	She is reading under the table.	She is reading under the table.^He is sitting on the table.^She is eating on the chair.
 			210002	soundonly_sentence					they will go fishing	They will go fishing.	The boy is fishing now.^They will take a bus.^They will go fishing.
 			210003	ordering_sentence	ßævØô¢ ·UUUUô âãè RUUUU× ×ð¢ Ú¹ð¢Ð	The sun is bright.It is very hot.During the day sun shines very bright and hot.				The sun is bright.^It is very hot.^During the day, the sun shines very bright and hot.	The sun is bright.^It is very hot.^During the day, the sun shines very bright and hot.
-			210004	listeningcomp_sentence	Abedi is a teacher.He has a wife. His wife is also a teacher.Both love teaching 	What does Abedi do?				He is a teacher.	He is a teacher.^He is a nurse.^He is a student.
+			210004	listeningcomp_sentence	Sahil is a teacher.He has a wife. His wife is also a teacher.Both love teaching 	What does Sahil do?				He is a teacher.	He is a teacher.^He is a nurse.^He is a student.
 			210005	listeningcomp_sentence	Cats are not large,but they are fast.Cats like to chase mice.They are fast when they chase mice.	Are cats fast?				Yes, they are.	Yes, they are.^No, they are not.^We do not know.
 											
 SpecialTest_21	2	210006, 210007, 210008, 210009, 210010									
 			210006	soundonly_sentence					Rabbits eat carrots.	Rabbits eat carrots.	Rabbits eat carrots.^Lions don't eat carrots.^People like carrots.
 			210007	soundonly_sentence					The boy wants to be a doctor.	The boy wants to be a doctor.	The boy wants to be a doctor.^The doctor is tall and kind.^The girl wants to be a teacher.
-			210008	ordering_sentence	ßævØô¢ ·UUUUô âãè RUUUU× ×ð¢ Ú¹ð¢Ð	Tim wanted to buy a donkey. He did not have enough money to buy a donkey. Tim was sad.				Tim wanted to buy a donkey.^He did not have enough money to buy a donkey.^Tim was sad.	Tim wanted to buy a donkey.^He did not have enough money to buy a donkey.^Tim was sad.
-			210009	listeningcomp_sentence	Abedi is a teacher.He has a wife. His wife is also a teacher.Both love teaching 	What does Abedi's wife do?				She is a teacher.	She is a teacher.^She is a nurse.^She is a student.
+			210008	ordering_sentence	ßævØô¢ ·UUUUô âãè RUUUU× ×ð¢ Ú¹ð¢Ð	Faraz wanted to buy a donkey. He did not have enough money to buy a donkey. Faraz was sad.				Faraz wanted to buy a donkey.^He did not have enough money to buy a donkey.^Faraz was sad.	Faraz wanted to buy a donkey.^He did not have enough money to buy a donkey.^Faraz was sad.
```

Comment:
Check whether you have the latest TSV file. If yes, replace the Chanakya text with proper Hindi text